diff -ru grib2/makefile grib2-new/makefile
--- grib2/makefile      2023-10-24 14:54:47.000000000 +0000
+++ grib2-new/makefile  2026-09-28 14:08:08.399073875 +0000
@@ -1041,7 +1041,7 @@
 	gunzip -f tmpproj4.tar.gz
 	tar -xvf tmpproj4.tar
 	rm tmpproj4.tar
-	cd ${proj4dir}  && export CFLAGS="${wCPPFLAGS}" &&  ./configure --disable-shared --prefix=${cwd} && ${MAKE} -j 1 check install
+	cd ${proj4dir}  && export CFLAGS="${wCPPFLAGS}" &&  ./configure --build=aarch64-unknown-linux-gnu --disable-shared --prefix=${cwd} && ${MAKE} -j 1 check install
 
 ${netcdf3dir}:
 	cp ${netcdf3src} tmpn.tar.gz
@@ -1050,7 +1050,7 @@
 	rm tmpn.tar
 
 ${netcdf3lib}:	${netcdf3dir}
-	cd ${netcdf3dir} && export CPPFLAGS="${netcdf3CPPFLAGS}" && ./configure --enable-c-only --prefix=${cwd} && ${MAKE} -j 1 check install
+	cd ${netcdf3dir} && export CPPFLAGS="${netcdf3CPPFLAGS}" && ./configure --build=aarch64-unknown-linux-gnu --enable-c-only --prefix=${cwd} && ${MAKE} -j 1 check install
 	cp ${netcdf3dir}/libsrc/netcdf.h ${cwd}/include
 
 ${netcdf4src}:
